src/Components/ActivationKey/ActivationKey.js

```diff
@@ -84,10 +73,13 @@ const ActivationKey = () => {
             </DescriptionListGroup>
           </LevelItem>
           <LevelItem className="pf-v5-u-mb-sm">
-            {user.rbacPermissions.canWriteActivationKeys ? (
-              <DeleteButton onClick={handleDeleteActivationKeyModalToggle} />
-            ) : (
-              <NoAccessPopover content={DeleteButton} />
+            {activationKey && (
+              <EditAndDeleteDropdown
+                name={id}
+                onClick={handleEditActivationKeyWizardToggle}
+                activationKey={activationKey}
+                releaseVersions={releaseVersions}
+              />
```

Comment:
Nitpick: Prefer `!isKeyLoading && <Component />` for readability

Issue: This still needs to check the rbac permissions to ensure a user can edit or delete a key